container/crio/handler.go

```diff
@@ -257,6 +257,8 @@ func (self *crioContainerHandler) getFsStats(stats *info.ContainerStats) error {
 	fsStat.BaseUsage = usage.BaseUsageBytes
 	fsStat.Usage = usage.TotalUsageBytes
 	fsStat.Inodes = usage.InodeUsage
+	fsStat.Available = usage.FreeBytes
```

Comment:
The limit is set up above based on the filesystem capacity.  Will that mean that Used + Free != Capacity?